src/NuGet.Core/NuGet.Frameworks/FrameworkNameProvider.cs

```diff
@@ -194,8 +211,9 @@ public string GetVersionString(string framework, Version version)
                     versionParts.Pop();
                 }
 
-                // Always use decimals and 2+ digits for dotnet, netstandard, netstandardapp,
-                // netcoreapp, or if any parts of the version are over 9 we need to use decimals
+                // Always use decimals and 2+ digits for netstandard and
+                // netcoreapp, or if any parts of the version are over 9
+                // we need to use decimals
```

Comment:
I wouldn't have noticed if you didn't reformat this comment, but I don't believe this is correct any longer.  net5.0 and above (net5Era) should also be using ".0" at the end.

Well, technically `net5` will work just fine, but we want to make sure any time we (or the SDK, or anything else calling NuGet's APIs) write the short name, we always use the ".0" version, so customers are used to that syntax for when "10.0" comes along. But I'm not sure if this is a non-net5era codepath, so maybe it's ok.

Reply:
net5.0 is netcoreapp everywhere internally, so the comment is accurate, and the behavior is correct.